docs/rgb_decomposition.md

```diff
@@ -0,0 +1,49 @@
+RGB Decomposition
+=================
+
+RGB products generated with HyP3 use both co-pol (VV or HH) and cross-pol (VH or HV) backscatter values to generate a color image from SAR data. This approach facilitates visual interpretation by decomposing the signals into surface scattering with some volume scattering (red band), volume scattering (green band), and surface scattering with very low volume scattering (blue band). 
+
+The dual-pol SAR dataset first undergoes radiometric terrain correction (RTC) to adjust for the distortions present in the images due to the side-looking geometry of SAR data acquisition. The RTC images are output in power scale, and the RGB values are calculated using a combination of the co-pol (CP) and cross-pol (XP) pixel values. The output RGB GeoTIFF is scaled so that the values for each band range from 1 to 255. Zero values are reserved for pixels with no data.
+
+### Interpreting the Images
+
+Calm water generally has very low returns in all polarizations, and areas with very low backscatter will appear blue. Desert landscapes have a backscatter signature similar to that of water, however, so care should be taken in interpretation of the blue color based on the location. 
+
+The green band is determined by the magnitude of the cross-pol returns, which indicates the extent of volume scattering. One of the most common volume scatterers is vegetation, and areas with high cross-pol returns appear more green in the decomposition. There are other volume scatterers, such as glacial ice, that may also appear as green, so interpretation will be site-dependent.
+
+The red channel is used to indicate areas that are neither water nor vegetation. Urban areas and other locations with relatively low levels of volume scattering will appear more red in this decomposition.
+
+### Decomposition Calculations
+
+Note that dual-pol image acquisition may provide either VV/VH returns or HH/HV returns, depending on the sensor mode. The same decomposition approach can be applied to either mode; use the co-pol (S<sub>CP</sub>) and cross-pol (S<sub>XP</sub>) pixel values available for the specific dataset. 
+
+To calculate the RGB values on a pixel-by-pixel basis, the data is first separated into surface (P<sub>r</sub>) and volume (P<sub>v</sub>) scattering components as follows:
+
+P<sub>r</sub> = S<sub>CP</sub> - 3 S<sub>XP</sub>
+
+P<sub>v</sub> = 4 S<sub>XP</sub>
```

Comment:
While not necessary, would this be easier to read if the Sxp variables being multiplied were bracketed? i.e. 3(Sxp) I waffled, but ended up going with the cleaner option.

Reply:
I like the cleaner option as well